.github/workflows/release_trigger.yml

```diff
@@ -12,13 +12,12 @@ jobs:
     steps:
       - name: Setup Environment
         id: setup
-        uses: actions/github-script@v6
+        uses: actions/github-script@v5
         with:
           script: |
-            if ('${{ github.event.action }}' === 'prereleased') {
+            if (context.payload.release.prerelease) {
               core.setOutput('environment', 'staging')
-            }
-            if ('${{ github.event.action }}' === 'released') {
+            } else {
```

Comment:
Like the original, these should be two If statements, not If/Else.

like the original, this should be be IF statements.
```
        with:
          script: |
            if (context.payload.release.prerelease) {
              core.setOutput('environment', 'staging')
            } 
            if (context.payload.release.release){
              core.setOutput('environment', 'prod')
            }
```